src/CompletedTaskList.js

```diff
@@ -2,12 +2,46 @@ import React from "react"
 
 import CreatedTask from "./CreatedTask"
 import Header from "./Header"
+import todosData from "./todosData"
 
-export default function CompletedTaskList(props) {
-  return (
-    <div>
-      <Header title="Completed Tasks" />
-      <CreatedTask task={props.task} completed="completed" />
-    </div>
-  )
-}
\ No newline at end of file
+class CompletedTaskList extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      todos: todosData,
+    }
+    this.handleClick = this.handleClick.bind(this)
+  }
+
+  handleClick(id) {
```

Comment:
This can be written in es6 syntax as a class function removing the need to bind it.
```suggestion
  handleClick = (id) => {
```

Reply:
Woot!  I vaguely remembered this syntax not being a thing anymore, but I couldn't remember what replaced it!